gsoc/gsoc2016/ideas_astropy.md

```diff
@@ -150,3 +150,31 @@ This project includes looking at other astronomy codes to see how they tackle th
 
 See https://github.com/OpenAstronomy/openastronomy.github.io/pull/27 for a discussion of some problems and possible solutions that will be addressed in this project.
 
+
+### Bridge sherpa and astropy fitting
+
+*Suggested Mentor(s):* D. Bourke, T. Aldcroft, H. M. Guenther
+
+*Difficulty:* Expert or better
+
+*Astronomy knowledge needed:* fitting functions and statistics
+
+*Programming skills:* Python, C, Cython
+
+#### Description
+
+
+Both astropy and Sherpa (https://github.com/sherpa/sherpa/) provide modelling and fitting capabilities; however, Sherpa's features are way
+more advanced. Sherpa provides far more build-in models, a larger choice of optimizers and a real variety
+of fit statistics.
+Unfortunately, Sherpa is less well known and (for partly historical reasons), the interface to do
```

Comment:
Suggest: "Unfortunately Sherpa is less well known and for historical reasons the object-oriented user interface is less polished than the functional state-based interface."  In other words you can do modern object-oriented code with Sherpa, it is just a bit difficult and not as well documented.